anvil/anvil-compiler/src/main/java/com/duckduckgo/anvil/compiler/ContributeToActivityStarterCodeGenerator.kt

```diff
@@ -68,14 +68,20 @@ class ContributeToActivityStarterCodeGenerator : CodeGenerator {
         val content = FileSpec.buildFile(generatedPackage, moduleClassName) {
             for (annotation in mapperAnnotations) {
                 val paramsType = annotation.paramsTypeOrNull()!!
-                addType(createMapperClass(paramsType, vmClass, module)).build()
+                val screenName = annotation.screenNameOrNull().orEmpty()
+                addType(createMapperClass(paramsType, screenName, vmClass, module)).build()
```

Comment:
Skip mapper generation if the screenName is null or blank
```suggestion
                if (screenName.isNullOrBlank() == false) {
                    addType(createMapperClass(paramsType, screenName, vmClass, module)).build()
                }
```